### PR TITLE
Adjust the margin to center the sunburst chart

### PR DESCRIPTION
**Pull request by javidhsueh**
Address issue #383

### CODE REVIEW

**Comment by javidhsueh** [comment]
screen shot 2017-04-13 at 11 00 26 am

**Comment by mcnuttandrew** [comment]
Looks like lint is failing

**Comment by mcnuttandrew** [comment]
I think this may also be an issue on the new Radial chart, will you take a look and see if this solution works there as well

**Comment by javidhsueh** [comment]
I actually notice the radial chart has the same issue as well. I'll create another PR

File: package.json [hunk 1]
```diff
@@ -43,9 +43,10 @@
     "deck.gl": "4.0.0",
     "deep-equal": "^1.0.1",
     "global": "^4.3.1",
-    "prop-types": "^15.5.6",
     "luma.gl": "^3.0.0-beta.11",
-    "react-motion": "^0.4.7"
+    "prop-types": "^15.5.6",
+    "react-motion": "^0.4.7",
```
Comment (mcnuttandrew) [inline package.json]: react-motion isn't already installed?

File: src/radial-chart/index.js [hunk 2]
```diff
@@ -92,6 +87,10 @@ function generateLabels(mappedData) {
   // could add force direction here to make sure the labels dont overlap
 }
 
+function getMaxRadius(width, height) {
```
Comment (mcnuttandrew) [inline src/radial-chart/index.js]: nit: jsdoc

**Comment by mcnuttandrew** [comment]
gotta run yarn remove-unpm-rfs to remove refs to unpm

File: package.json (hunk 1 above)
Comment (javidhsueh) [inline package.json]: I think it's just to reorder them

Comment (mcnuttandrew) [inline package.json]: whoops totally didn't see dat

**Pull request merged**